x/evm/statedb/statedb.go

```diff
@@ -53,9 +51,18 @@ var _ vm.StateDB = &StateDB{}
 // * Contracts
 // * Accounts
 type StateDB struct {
-	keeper   Keeper
-	ctx      sdk.Context
-	cacheCtx sdk.Context
+	keeper Keeper
+	// origCtx is the context passed in by the caller
+	origCtx sdk.Context
+	// ctx is a branched context on top of the context the caller passes in
+	ctx sdk.Context
+	// cacheMS caches the `ctx.MultiStore()` to avoid type assertions all the time.
```

Comment:
```suggestion
	// cacheMS caches the `ctx.MultiStore()` to avoid type assertions all the time, `ctx.MultiStore()` is not modified during the whole time which is evident by `ctx.WithMultiStore` is not called after statedb constructed.
```